specificatie/openapi.yaml

```diff
@@ -197,11 +332,90 @@ paths:
+        '401':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/401"
+        '403':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/403"
+        '404':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/404"
+        '406':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/406"
+        '500':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/500"
+        '501':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/501"
+        '503':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/503"
+        'default':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/default"
+      tags:
+        - BRP historie bevragen
 components:
   parameters:
     burgerservicenummer:
       $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/v1.2.0/specificatie/openapi.yaml#/components/parameters/burgerservicenummer"
+#Is het verwijzen tussen verschillende API-specificaties voor dergelijke definities nog wel verstandig ? We raken volgens mij het overzicht kwijt.
   schemas:
+    IngeschrevenPersoonQuery:
+      type: "object"
+      properties:
+        burgerservicenummer:
+          description: |
+                        Uniek persoonsnummer
+          example: 999993653
+          type: "string"
+          pattern: "^[0-9]{9}$"
+        fields:
+          type: string
+          description: "Hiermee kun je de inhoud van de resource naar behoefte aanpassen door een door komma's gescheiden lijst van property namen op te geven. Bij opgave van niet-bestaande properties wordt een 400 Bad Request teruggegeven. De fiels parameter is verplicht vanwege dataminimalisatie overwegingen. Zie [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/fields.feature)"
+          pattern: "^[a-zA-Z.,_]+$"
+        peildatum:
+          description: |
+                        De datum waarop de resource wordt opgevraagd.
+
+          type: "string"
+          format: date
+          example: "2018-09-09"
+        datumVan:
+          description: |
+                        De begindatum van de periode waarover de resource wordt opgevraagd.
```

Comment:
kunnen we "resource" hier niet vervangen door "historie"?

Reply:
Wat mij betreft geen probleem. Dat betekent wel dat we niet meer kunnen hergebruiken uit de common. @CathyDingemanse wat vind jij daar van ?

Reply:
wordt datumVan ook gebruikt voor iets anders dan het vragen van historie?